tests/facility.model.test.ts

```diff
@@ -19,7 +19,7 @@ describe("Facility Model Test", () => {
     const facilityData = {
       Name: "Facility One",
       Category: "General",
-      Tags: "Clean, Spacious",
+      Tags: ["Clean", "Spacious", "Family-friendly"],
```

Comment:
change to use the pre set tags -- Wheelchair accessible, Baby changing, regularly cleaned, and/or gender neutral

Reply:
Really unnecessary for the test but fine. fixed.